cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/AdaptTemplatesCommand.java

```diff
@@ -30,6 +31,12 @@ public class AdaptTemplatesCommand extends CommandCommons {
    */
   private static Logger LOG = LoggerFactory.getLogger(CobiGenCLI.class);
 
+  /**
+   * If this options is enabled, all templates are unpacked.
+   */
+  @Option(names = { "--all" }, negatable = true, description = MessagesConstants.UPDATE_ALL_DESCRIPTION)
```

Comment:
The text of `MessagesConstants.UPDATE_ALL_DESCRIPTION` does not match in this context. Create an own constant for this